### DISCUSSION TITLE
Underlying theoretic papers behind MFGP-BO

### DISCUSSION THREAD

**Discussion by TimLK** [meta-pytorch/botorch#1392] [Q&A] [answered]
Hi, I am quite interested in Multi-fidelity Gaussian Process and Bayesian Optimization but relatively new to this field. I have two simple questions:
1. What published papers that the `SingleTaskMultiFidelityGP` and `MultiFidelityKnowledgeGradient` are created to be based. (I really want to know this)
2. Why does BoTorch only create KG for MFBO? There seems to be several other acquisition functions for MFGP-BO, some of which seems to show higher performance.
Thank you very much in advance!!

**Comment by saitcakmak** [comment, marked as answer]
Hi @TimLK.

> What published papers that the SingleTaskMultiFidelityGP and MultiFidelityKnowledgeGradient are created to be based. 

We generally link to the relevant papers in the source code. In this case, you can find the relevant papers in the [tutorial](https://botorch.org/tutorials/discrete_multi_fidelity_bo) or in the [code](https://github.com/pytorch/botorch/blob/main/botorch/models/gp_regression_fidelity.py#L21-L23)

> Why does BoTorch only create KG for MFBO?

We generally implement new methods on an as-needed basis, and also review PRs from external users. In the case of MFBO, I think there are also two max-value entropy search based acquisition functions implemented [here](https://github.com/pytorch/botorch/blob/main/botorch/acquisition/max_value_entropy_search.py#L652) and [here](https://github.com/pytorch/botorch/blob/main/botorch/acquisition/max_value_entropy_search.py#L801), as well as a multi-objective multi-fidelity acquisition function based on hypervolume improvement implemented [here](https://github.com/pytorch/botorch/blob/main/botorch/acquisition/multi_objective/multi_fidelity.py)

**Reply by TimLK** [reply]
Thanks very much for your prompt reply. Sorry because I did not notice the other two entropy search-based acquisition functions as they do not appear in the tutorials.
Another small side question, which was not attended in #947, why the kernel LinearTruncatedFidelityKernel is used while does not appear in the referenced paper nearside the tutorials. Is there any reference about it?

**Reply by saitcakmak** [reply]
>  why the kernel LinearTruncatedFidelityKernel is used

I don't have any context on the kernel choice. I presume it must have been tested on some internal applications and found to have better performance.

**Reply by TimLK** [reply]
Thanks again, I appreciate it a lot!

**Reply by Balandat** [reply]
I think @VilockLi originally implemented this and may have more context.
